data/blog/electron-11-0.md

```diff
@@ -0,0 +1,61 @@
+
+See the [11.0.0 release notes](https://github.com/electron/electron/releases/tag/v11.0.0) for a full list of new features and changes.
+
+## Breaking Changes
+
+* TBD
+
+More information about these and future changes can be found on the [Planned Breaking Changes](https://github.com/electron/electron/blob/master/docs/breaking-changes.md) page.
+
+## API Changes
+
+To make upgrading easier for our app developers, Electron 11 was a quiet release. No Electron API changes or depreciations were made.
+
+## End of Support for 8.x.y
+
+Electron 8.x.y has reached end-of-support as per the project's [support policy](https://electronjs.org/docs/tutorial/support#supported-versions). Developers and applications are encouraged to upgrade to a newer version of Electron.
+
+## What's Next
+
+In the short term, you can expect the team to continue to focus on keeping up with the development of the major components that make up Electron, including Chromium, Node, and V8. Although we are careful not to make promises about release dates, our plan is release new major versions of Electron with new versions of those components approximately quarterly. The [tentative 12.0.0 schedule](https://electronjs.org/docs/tutorial/electron-timelines) maps out key dates in the Electron 12.0 development life cycle. Also, [see our versioning document](https://electronjs.org/docs/tutorial/electron-versioning) for more detailed information about versioning in Electron.
+
+For information on planned breaking changes in upcoming versions of Electron, [see our Planned Breaking Changes doc](https://github.com/electron/electron/blob/master/docs/breaking-changes.md).
+
+### Continued Work for Deprecation of `remote` Module (in Electron 12)
+We started work to remove the remote module in [Electron 9](https://www.electronjs.org/blog/electron-9-0) and we're continuing plans to remove the `remote` module. In Electron 12, we plan to continue refactor work for implementing [WeakRef](https://v8.dev/features/weak-references) as we have done in Electron 11. Please read and follow [this issue](https://github.com/electron/electron/issues/21408) for full plans and details for deprecation.
+
+### Final Step for Requiring Native Node Modules to be Context Aware or N-API (in Electron 12)
+From Electron 6 onwards, we've been laying the groundwork to require [native Node modules](https://nodejs.org/api/addons.html) loaded in the renderer process to be either [N-API](https://nodejs.org/api/n-api.html) or [Context Aware](https://nodejs.org/api/addons.html#addons_context_aware_addons). Enforcing this change allows for stronger security, faster performance, and reduced maintenance workload. The final step of this plan is to remove the ability to disable render process reuse in Electron 12. Read [this issue](https://github.com/electron/electron/issues/18397) for full details including the proposed timeline.
```

Comment:
@MarshallOfSound the issue https://github.com/electron/electron/issues/18397 still states Electron 11 for `Remove the ability to change app.allowRendererProcessReuse in Electron 11`
Can you please update this to Electron 12?

Reply:
Ah, I should probably also update the Electron 10 blog post to reflect this too, I can do that in this PR